### PR TITLE
Adding the functionality for activate, deactivate and delete Judgers

### PR DESCRIPTION
**Pull request by abdallaahmedsd**

### CODE REVIEW

**Comment by Musab991** [comment]
Nice work!

File: AADL/Judgers/frmJudgersList.cs [hunk 1]
```diff
@@ -29,15 +29,15 @@ private void _FillDataGridView(DataTable dtJudgers)
...
 
-                dgvJudgers.Columns["JudgerID"].HeaderText = "الرقم التعريفي كمحكم";
-                dgvJudgers.Columns["PractitionerID"].HeaderText = "الرقم التعريفي كمزوال";
+                dgvJudgers.Columns["JudgerID"].HeaderText = "الرقم التعريفي";
                 dgvJudgers.Columns["FullName"].HeaderText = "الاسم رباعي";
                 dgvJudgers.Columns["Phone"].HeaderText = "رقم الهاتف";
                 dgvJudgers.Columns["Gender"].HeaderText = "النوع";
                 dgvJudgers.Columns["CountryName"].HeaderText = "الدولة";
                 dgvJudgers.Columns["CityName"].HeaderText = "المدينة";
                 dgvJudgers.Columns["Address"].HeaderText = "العنوان";
```
Comment (Musab991) [inline AADL/Judgers/frmJudgersList.cs:38]: Address would be better if replaced with email , it's more used .

**Review by Musab991 — COMMENTED** [review]
Mashallah I liked your way in creating judger list

**Comment by abdallaahmedsd** [comment]
Thank you 😁😁

**Pull request merged**